### PR TITLE
book: update chapter landing pages.

### LINKED ISSUE #1663: Updating Chapter Landing Pages
### Summary

The new chapter template recommends that the `How This is Useful/Why This is Helpful` section of chapter landing pages should be titled as `Motivation and Background`. This issue curates a list of all chapters that still use the old title and welcomes contributors to update them to follow the template's recommendations.

### What needs to be done?

Pick a chapter landing page from the list (tick it off from the list) below and change the title of the `How This is Useful/Why This is Helpful` section to `Motivation and Background`:

- [x] [Open Research](https://the-turing-way.netlify.app/reproducible-research/open.html)
- [x] [Licensing](https://the-turing-way.netlify.app/reproducible-research/licensing.html)
- [x] [Research Data Management](https://the-turing-way.netlify.app/reproducible-research/rdm.html)
- [x] [Reproducible Environments](https://the-turing-way.netlify.app/reproducible-research/renv.html)
- [x] [BinderHub](https://the-turing-way.netlify.app/reproducible-research/binderhub.html)
- [x] [Code Testing](https://the-turing-way.netlify.app/reproducible-research/testing.html)
- [x] [Continuous Integration](https://the-turing-way.netlify.app/reproducible-research/ci.html)
- ~~[ ] [Credit for Reproducible Research](https://the-turing-way.netlify.app/reproducible-research/credit.html)~~  **404- Note that this page no longer exists.**
- [x] [Case Studies](https://the-turing-way.netlify.app/reproducible-research/case-studies.html)
- [x] [File Naming Convention](https://the-turing-way.netlify.app/project-design/filenaming.html)
- [x] [Community Communications for Open Source Projects](https://the-turing-way.netlify.app/communication/os-comms.html)
- ~~[ ] [Collaborating on GitHub/GitLab](https://the-turing-way.netlify.app/collaboration/github-collaboration.html)~~ **404- Note that this page no longer exists.**
- [x] [Ethical Decisions in Preclinical Research](https://the-turing-way.netlify.app/ethical-research/preclinical-research.html)

### Who can help?

* Anyone interested in keeping the Turing Way consistent. We especially welcome new contributors to try their hand at this

---

### Updates

*Lorem ipsum dolor sit amet, consectetur adipiscing.*

### PR DESCRIPTION
**Pull request by TheAbstract**
### Summary

Updates chapter landing pages.

Fixes #1663

### List of changes proposed in this PR (pull-request)

- update chapter landing pages.

### What should a reviewer concentrate their feedback on?

The following pages no longer exist as mentioned in the [issue](https://github.com/alan-turing-institute/the-turing-way/issues/1663):
- [Credit for Reproducible Research](https://the-turing-way.netlify.app/reproducible-research/credit.html)
- [Collaborating on GitHub/GitLab](https://the-turing-way.netlify.app/collaboration/github-collaboration.html)

- [ ] Everything looks ok?

### Acknowledging contributors

- [ ] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/main/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/main/README.md#contributors) in the README file:

### CODE REVIEW

**Comment by TheAbstract** [comment]
@all-contributors please add @TheAbstract for content

File: .vscode/settings.json
[the remark this answers is not in the document]
Comment (TheAbstract) [reply .vscode/settings.json:1]: 🤦🏾 accident, wasn't on my machine so I used the GitHub editor thing when doing this. Will fix soon, I've never even used vscode 😀.

**Review by da5nsy — APPROVED** [review]
LGTM! 🎉

File: .vscode/settings.json
[the remark this answers is not in the document]
Comment (da5nsy) [reply .vscode/settings.json:1]: No worries at all! I've been there!

**Pull request merged**